### PR TITLE
Fix Button affixes - part 2

### PR DESCRIPTION
**Pull request by jonathanadler**
This reverts commit 8a18c7b6edb547d7849db63cde2fd8bf98e68728.

# ✨ Pull Request

### 💁 Description
Adding back the fix for the affix icons of the button

...

### 👀  PR is ready for review 

- [x] Ready for review

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_579/

File: src/components/Button/Button.st.css [hunk 1]
```diff
@@ -355,3 +358,55 @@
...
+.root:priority(basic) .affixIcon svg path,
+.root:priority(primary) .affixIcon svg path {
+    stroke: value(PrimaryTextColor);
+}
+
+.root:hover .affixIcon svg path,
+.root:priority(basic):hover .affixIcon svg path,
+.root:priority(primary):hover .affixIcon svg path {
+    stroke: applyOpacity(value(PrimaryTextColor), 0.7);
+}
```
Comment (sivanharel) [inline src/components/Button/Button.st.css:377]: Here also, could we just define:
```
.root:hover .affixIcon svg path {
       stroke: applyOpacity(value(PrimaryTextColor), 0.7);
}
```

and then override only the `secondary` and the `basicSecondary` priority ?

File: src/components/Button/Button.st.css [hunk 2]
```diff
@@ -355,3 +358,55 @@
 .root:upgrade:size(large):mobile, .root:priority(primary):upgrade:size(large):mobile, .root:priority(basic):upgrade:size(large):mobile{
     padding: calculate(string(-), 17px, value(secondaryBorderWidth)) 16px;
 }
+
+.root .affixIcon svg path {
+    fill: none;
+    transition: stroke .2s ease-in-out;
```
Comment (sivanharel) [inline src/components/Button/Button.st.css:364]: Couldn't we just add the `stroke: value(PrimaryTextColor);` here and remove the:

```
.root .affixIcon svg path,
.root:priority(basic) .affixIcon svg path,
.root:priority(primary) .affixIcon svg path {
    stroke: value(PrimaryTextColor);
}

```

**Review by sivanharel — APPROVED** [review]
Awesome!
Just a very small comment.

File: src/components/Button/Button.st.css (hunk 1 above)
Comment (jonathanadler) [reply src/components/Button/Button.st.css:377]: Good point 🤔

**Pull request merged**